### PR TITLE
Add VC 6.x support to generate_html5_console.py

### PR DESCRIPTION
**Pull request by derchrisuk**
Modified the script to generate URL for 5.x and 6.x vSphere

### CODE REVIEW

File: samples/generate_html5_console.py [hunk 1]
```diff
@@ -103,13 +110,27 @@ def main():
                                              vc_cert)
     vc_fingerprint = vc_pem.digest('sha1')
 
+    if vcenter_version <= 5:
+        console_port = '7331'
+        console_url = "http://" + args.host + ":" + console_port \
```
Comment (tianhao64) [inline samples/generate_html5_console.py]: Thanks @derchrisuk for the fix. Can we use the str.format() to format the string here?

File: samples/generate_html5_console.py [hunk 2]
```diff
@@ -69,11 +69,15 @@ def main():
 
     args = get_args()
 
+    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+    context.verify_mode = ssl.CERT_NONE
+
     try:
         si = SmartConnect(host=args.host,
```
Comment (tianhao64) [inline samples/generate_html5_console.py]: SmartConnectNoSSL is an alternative if you want to skip the cert check. You don't need to create the context if you use that.

File: samples/generate_html5_console.py (hunk 1 above)
Comment (derchrisuk) [inline samples/generate_html5_console.py]: I can have a look.

File: samples/generate_html5_console.py (hunk 2 above)
Comment (derchrisuk) [inline samples/generate_html5_console.py]: I never got it to work, none of the samples which were using it worked for me.

Comment (tianhao64) [inline samples/generate_html5_console.py]: It was introduced recently. Which version of pyvmomi do you have?

Comment (tianhao64) [inline samples/generate_html5_console.py]: It's part of pyvmomi v6.0.0.2016.6 release.

Comment (zhenmie365) [inline samples/generate_html5_console.py]: Thanks! It works.
But I had to login in the web Client using user "administrator@vsphere.local" and password "xxxx..."  to get certificate, at the first time of accessing html5 console in the code using SmartConnectNoSSL on vsphere 6.x and pyvmomi 6.5 release.Is this right?
And, there is bug in line 113 "if vcenter_version <= 5:" as vcenter_version is {str}. It always comes out "false" whenever running code in vsphere 5.x environment.

Comment (zhenmie365) [inline samples/generate_html5_console.py]: the vcenter_version is like "5.0.0" when running code in vsphere 5.0.0

Comment (zhenmie365) [inline samples/generate_html5_console.py]: And there is problem to access html console of vm with name containing chinese, as the parameter 'cfgFile' is converted like this '%2Fvmfs%2Fvolumes%2F58c1142a-68caa49c-adea-005056b93c6c%2F%E6%B5%8B%E8%AF%951lzy%2F%E6%B5%8B%E8%AF%951lzy.vmx' when vm name is "测试1lzy".

**Comment by zhangjianpinghik** [comment]
a error occured when i run the samples：“generate_html5_console.py” 

vc_cert = ssl.get_server_certificate((self.host, int(self.port)))
File "/usr/lib/python2.7/ssl.py", line 548, in get_server_certificate
s.connect(addr)
File "/usr/lib/python2.7/ssl.py", line 433, in connect
self._real_connect(addr, False)
File "/usr/lib/python2.7/ssl.py", line 423, in _real_connect
self.do_handshake()
File "/usr/lib/python2.7/ssl.py", line 405, in do_handshake
self._sslobj.do_handshake()
ssl.SSLError: [Errno 8] _ssl.c:510: EOF occurred in violation of protocol

**Comment by pgbidkar** [comment]
Closing PR due to inactivity from long time. 
Please open new PR if changes are still relevant 

Thanks

**Pull request closed without merging**